examples/standalone-data-generator/configure.py

```diff
@@ -0,0 +1,73 @@
+"""
+Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License"). You may not use this file except in compliance
+with the License. A copy of the License is located at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+or in the 'license' file accompanying this file. This file is distributed on an 'AS IS' BASIS, WITHOUT WARRANTIES
+OR CONDITIONS OF ANY KIND, express or implied. See the License for the specific language governing permissions
+and limitations under the License.
+"""
+import json
+import enums
+
+APP_TYPE = enums.Application.NotePad
```

Comment:
add commented code for highlighting how switching to Shopping

Reply:
added.